tests/integration/test_emr.py

```diff
@@ -11,22 +11,23 @@
 # ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
 from tests import unittest
+import pytest
 
 import botocore.session
 from botocore.paginate import PageIterator
 from botocore.exceptions import OperationNotPageableError
 
 
-def test_emr_endpoints_work_with_py26():
+@pytest.mark.parametrize('region',['us-east-1', 'us-west-2', 'ap-northeast-1',
```

Comment:
Might be easier to just pull this list out as a `EMR_REGIONS` variable at the module level or something.